src/test/java/org/springframework/samples/petclinic/owner/PetTests.java

```diff
@@ -0,0 +1,81 @@
+package org.springframework.samples.petclinic.owner;
+
+import static org.junit.Assert.*;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.samples.petclinic.visit.Visit;
+
+import java.util.List;
+import java.util.Set;
+import java.util.LinkedHashSet;
+import java.time.*;
+import java.util.Date;
+
+public class PetTests {
+
+	private Pet pet;
+	private Date birthDate;
+
+	private void establishCurrentDateForTesting() {
```

Comment:
Is there a specific reason as to why `timePoint`, `localDate`, and `birthDate` are initialized in its own private method rather than in `testSetUp`?

Reply:
Not particularly, just felt that since I was only using birthDate in one of the tests, I went about it this way. Will refactor it so that this is in testSetUp.